SwiftCarousel.swift

```diff
@@ -202,6 +200,9 @@ public class SwiftCarousel: UIView {
     override public func layoutSubviews() {
         super.layoutSubviews()
         
+        guard (scrollView.frame.width > 0 &&
+            scrollView.frame.height > 0)  else { return }
+        
```

Comment:
in my case there is some cycles when the frame is 0 probably because of some temporary constraints in the UIStackView, so we skip any computation for those layout passes